src/Stryker.Core/Stryker.Core.UnitTest/Options/Inputs/ProjectVersionInputTests.cs

```diff
@@ -0,0 +1,26 @@
+using Shouldly;
+using Stryker.Core.Exceptions;
+using Stryker.Core.Options.Inputs;
+using Stryker.Core.Reporters;
+using Xunit;
+
+namespace Stryker.Core.UnitTest.Options.Inputs
+{
+    public class ProjectVersionInputTests
+    {
+        [Theory]
+        [InlineData(null)]
+        [InlineData("")]
+        public void ProjectVersionCannotBeEmpty(string value)
+        {
+            var input = new ProjectVersionInput { };
+            input.SuppliedInput = value;
+
+            var exception = Should.Throw<StrykerInputException>(() => {
```

Comment:
Just a thought, maybe it's worth checking the inner exception from time to time, as this StrykerInputException is thrown very often in a lot of places :)